executor/index_lookup_merge_join.go

```diff
@@ -30,6 +25,9 @@ import (
 	"github.com/pingcap/tidb/util/memory"
 	"github.com/pingcap/tidb/util/ranger"
 	"github.com/pingcap/tidb/util/stringutil"
+	"sort"
+	"sync"
+	"sync/atomic"
```

Comment:
Please separate third-party library and standard library.

Reply:
Done.